### PR TITLE
feat: distribute PSM Charter Invitatitons

### PR DESCRIPTION
**Pull request by Chris-Hibbert**
refs: #6110

## Description

Distribute PSM Charter invitations to committee members in bootstrap

### Security Considerations

The Charter, and the ability to instigate votes, should only be accessible to members of the committee.

### Documentation Considerations

[Eventually] we should write up how governance works through the smartWallet. This is a small aspect of that.

### Testing Considerations

A manual End-2-End is coming shortly.

### CODE REVIEW

File: packages/inter-protocol/src/proposals/startPSM.js [hunk 1]
```diff
@@ -339,6 +340,68 @@ export const PSM_GOV_MANIFEST = {
...
+   * @param {[string, Promise<Invitation>][]} addrInvitations
+   */
+  const distributeInvitations = async addrInvitations => {
+    await Promise.all(
+      addrInvitations.map(async ([addr, invitationP]) => {
+        const [voterInvitation, charterMemberInvitation] = await Promise.all([
+          invitationP,
+          E(creatorFacet).makeCharterMemberInvitation(),
+        ]);
+        console.log('@@@sending charter, voting invitations to', addr);
```
Comment (dckc) [inline packages/inter-protocol/src/proposals/startPSM.js]:
```suggestion
        console.log('sending charter, voting invitations to', addr);
```

That `@@@` thingy is my "I have to run to a meeting now; make sure I don't ship this half-baked code" marker. Let's take it out. Or we can drop the log message altogether, but I think maybe this is worth keeping.

File: packages/inter-protocol/src/proposals/startPSM.js [hunk 2]
```diff
@@ -339,6 +340,68 @@ export const PSM_GOV_MANIFEST = {
...
+        await reserveThenDeposit(
+          `econ committee member ${addr}`,
+          namesByAddressAdmin,
+          addr,
+          [voterInvitation, charterMemberInvitation],
+        );
+        console.log('@@@sent charter, voting invitations to', addr);
```
Comment (dckc) [inline packages/inter-protocol/src/proposals/startPSM.js]:
```suggestion
        console.log('sent charter, voting invitations to', addr);
```

File: packages/vats/src/core/boot-psm.js [hunk 3]
```diff
@@ -173,10 +175,13 @@ export const buildRootObject = (vatPowers, vatParameters) => {
       startEconomicCommittee(powersFor('startEconomicCommittee'), {
         options: {
           econCommitteeOptions: {
-            committeeSize: economicCommitteeAddresses.length,
+            committeeSize: Object.values(economicCommitteeAddresses).length,
           },
         },
       }),
+      invitePSMCommitteeMembers(powersFor('invitePSMCommitteeMembers'), {
```
Comment (dckc) [inline packages/vats/src/core/boot-psm.js:182]: :+1:

**Review by dckc — APPROVED** [review]
good stuff!

File: packages/inter-protocol/src/proposals/startPSM.js (hunk 1 above)
Comment (Chris-Hibbert) [reply packages/inter-protocol/src/proposals/startPSM.js]: done

File: packages/inter-protocol/src/proposals/startPSM.js (hunk 2 above)
Comment (Chris-Hibbert) [reply packages/inter-protocol/src/proposals/startPSM.js]: done

**Pull request merged**